index.js

```diff
@@ -4,7 +4,7 @@ import './index.css';
 const savedState = JSON.parse(localStorage.getItem('savedState'));
 
 const _selectedOrgGags =
-  savedState?.selectedOrgGags || Array.from({ length: 4 }, () => null);
+  savedState?.selectedOrgGags || Array.from({ length: 5 }, () => null);
```

Comment:
Why was this changed from 4 to 5?

Reply:
Oh, I thought it was related to the localStorage savedState code, but now that I look at it, it's about organic gags, I'll reverse that change.